db/schema.rb

```diff
@@ -258,13 +270,14 @@
 
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
   add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
-  add_foreign_key "adopter_accounts", "users"
-  add_foreign_key "adopter_applications", "adopter_accounts"
+  add_foreign_key "adopter_applications", "adopter_accounts", column: "adopter_account_id"
```

Comment:
Down here, need to make sure adopter_accounts becomes adopter_foster_accounts sand remove references to foster_accounts